lib/shipit/predictive_build_commands.rb

```diff
@@ -58,6 +58,10 @@ def git_get_commits_in_range(from, to)
       git('rev-list', "#{from}..#{to}",'--first-parent', chdir: @chdir)
     end
 
+    def git_delete_branch(branch)
```

Comment:
if you're sending `chdir` as an input parameter shouldn't it be part of the signature?

Reply:
it's `@chdir` the `@` indicates that this is a class member and not a variable.
See other methods above for usage example

Reply:
in this case, why are you sending `chdir` when calling `git_delete_branch`?

Reply:
I'm not sure if this is needed, I assume we need to specify the working directory...
I just took this from other examples in the file